### PR TITLE
Location finding

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/ussdapp.js [hunk 1]
```diff
@@ -367,6 +370,193 @@ go.app = function() {
...
+
+
+    // REFUGEE MENU STATES
+
+        // 024
+        self.add('state_024', function(name) {
+            return new ChoiceState(name, {
+                question: $('Select an option'),
+                choices: [
+                    new Choice('state_locate_me', $("Find nearest SService"))
```
Comment (imsickofmaps) [inline src/ussdapp.js:550]: Typo `SService`

Comment (gsvr) [inline src/ussdapp.js:550]: The content is not available yet so it'll have to change later in any case, but that's short for Support Service. It's just placeholder text for now.

**Comment by imsickofmaps** [comment]
:+1: We need to see this working in the "real world"

**Pull request merged**